README.adoc

```diff
@@ -352,6 +352,23 @@ refute('rubocop-minitest'.kind_of?(String))
 refute_kind_of(String, 'rubocop-minitest')
 ----
 
+=== Global Expectations [[global-expectations]]
```

Comment:
Btw, you should also introduce an `== Expectations` heading, as all the rules so far has been about assertions. Sadly we haven't documented any best practices about Minitest expectations yet.

Reply:
@abhaynikam Definitely! We'd appreciate any help with that!